docs/DeveloperGuide.md

```diff
@@ -434,19 +458,24 @@ An instance of `CommandResult` is then created which contains the message and in
-|----------|---------|-----------------------------------|------------------------------------------------|
-| `* * *`  | user    | view all the available commands   | I know how to use the app                      |
-| `* * *`  | user    | add a new applicant               | track the the progress of all applicants       |
-| `* * *`  | user    | edit an applicant descriptor      | maintain an updated database of all applicants |
-| `* * *`  | user    | view the full list of applicants  | access important applicant information         |
-| `* * *`  | user    | delete an applicant entry         | only track valid applicants                    |
-| `* * *`  | user    | add an interview for an applicant | plan screenings                                |
-| `* * *`  | user    | store data locally                | use it on a daily basis consistently           |
-| `* *`    | user    | find a specific applicant         | access the applicant's information quickly     |
-| `* *`    | user    | sort applicants by a descriptor   | find relevant applicants quickly               |
-| `* *`    | user    | filter applicants by a descriptor | find relevant applicants quickly               |
-| `* *`    | user    | purge all existing data           | remove sample data and populate real data      |
+| Priority | As a …​ | I want to …​                                               | So that I can…​                              |
+|----------|---------|------------------------------------------------------------|----------------------------------------------|
+| `* * *`  | user    | view all the available commands                            | know how to use the app                      |
+| `* * *`  | user    | add a new applicant                                        | track the the progress of all applicants     |
+| `* * *`  | user    | edit an applicant descriptor                               | maintain an updated database of all applicants |
+| `* * *`  | user    | view the full list of applicants                           | access important applicant information       |
+| `* * *`  | user    | delete an applicant entry                                  | only track valid applicants                  |
+| `* * *`  | user    | add an interview for an applicant                          | plan screenings                              |
+| `* * *`  | user    | store data locally                                         | use it on a daily basis consistently         |
+| `* *`    | user    | find a specific applicant                                  | access the applicant's information quickly   |
+| `* *`    | user    | sort applicants by a descriptor                            | find relevant applicants quickly             |
+| `* *`    | user    | filter applicants by a descriptor                          | find relevant applicants quickly             |
```

Comment:
The benefit of sort and filter is the same. Maybe change the benefit of filter to something else like "only look at applicants of a specific category"